src/osal/lv_freertos.c

```diff
@@ -318,6 +318,52 @@ lv_result_t lv_thread_sync_delete(lv_thread_sync_t * pxCond)
     return LV_RESULT_OK;
 }
 
+lv_result_t lv_thread_sync_signal_isr(lv_thread_sync_t * sync)
+{
+#if USE_FREERTOS_TASK_NOTIFY
+    /* Send a notification to the task waiting. */
+    vTaskNotifyGiveFromISR(pxCond->xTaskToNotify, NULL);
```

Comment:
Don't use NULL.
Take care of xHigherPriorityTaskWoken --> I already copy-paste an example of usage.

Reply:
Applied the suggestion, also for the other branch. Hopefully correct since I am not an Free RTOS expert :^)